framework/source/class/qx/ui/window/Window.js

```diff
@@ -929,6 +1055,40 @@ qx.Class.define("qx.ui.window.Window",
       }
     },
 
+    // overridden
+    _applyCenterWhen : function(value, old)
+    {
+      var             parent;
+
+      // Remove prior listener for centering on appear
+      if (this.__centeringAppearId !== null) {
+        this.removeListenerById(this.__centeringAppearId);
+        this.__centeringAppearId = null;
+      }
```

Comment:
At least the listener on *this* could be omitted. Just calling *this.removeListener("appear", this.center, this)* is enough, as it doesn't fail when it does not exist. But - well, that's a matter of taste, of course.